apps/browser/src/connectors/sso.ts

```diff
@@ -0,0 +1,26 @@
+﻿document.addEventListener("DOMContentLoaded", () => {
+  const lastPass = getQsParam("lp");
+  const code = getQsParam("code");
+  const state = getQsParam("state");
+  if (lastPass === "1") {
+    window.location.href =
+      "/popup/index.html?uilocation=popout#/import?code=" + code + "&state=" + state;
+  }
+});
+
+function getQsParam(name: string) {
+  const url = window.location.href;
+  // eslint-disable-next-line
+  name = name.replace(/[\[\]]/g, "\\$&");
```

Comment:
```suggestion
  name = name.replace(/[[\]]/g, "\\$&");
```

Based on the eslint error here - `ESLint: Unnecessary escape character: \[.(no-useless-escape)` it seems like the applying the above suggestion would allow you to avoid disabling the linter. 

Does the result somehow differ without the escape character?

Reply:
I am not sure. I just copied this code from the connectors on the web vault. Regex is not something I try to understand :D

Reply:
I'm in the same boat, hah. I usually use [cheat tools](https://regex101.com/) to help me comprehend it. 

If this was pulled from elsewhere in the codebase, then it's not a blocker imo.